tensorflow/core/common_runtime/copy_tensor.h

```diff
@@ -55,8 +55,8 @@ class CopyTensor {
    public:
     Registration(DeviceType sender_device_type, DeviceType receiver_device_type,
                  CopyFunction copy_function) {
-      TF_QCHECK_OK(
-          Register(sender_device_type, receiver_device_type, copy_function));
+      TF_QCHECK_OK(Register(sender_device_type, receiver_device_type,
+                            copy_function, /*is_pluggable_device*/ false));
```

Comment:
Nit: Please comment in this format so ClangTidy can [help check the parameter's name](https://g3doc.corp.google.com/devtools/cymbal/clang_tidy/g3doc/checks/bugprone-argument-comment.md?cl=head)
```suggestion
                            copy_function, /*is_pluggable_device=*/ false));
```

Reply:
done